test/jdk/jdk/nio/zipfs/ZipFSTester.java

```diff
@@ -1079,4 +1082,23 @@ static Path getPathWithParents(FileSystem fs, String name)
             mkdirs(parent);
         return path;
     }
+
+    static void test8299864(FileSystem fs)  {
```

Comment:
Please add a comment explaining the test and change the method name as would prefer to have names that are a bit more descriptive than a bug number.

I realize some of the existing methods follow the same naming as you are proposing, but we are trying to avoid this and make the tests more descriptive going forward